source/monitoring/bucket-notifications/publish-events-to-amqp.rst

```diff
@@ -0,0 +1,391 @@
+        any existing settings for that endpoint. Use 
+        :mc-cmd:`mc admin config get notify_amqp <mc admin config get>` to
+        review the currently configured AMQP endpoints on the MinIO deployment.
+
+      - Replace ``<ENDPOINT>`` with the URL of the AMQP service endpoint.
+        For example:
+
+        ``amqp://user:password@hostname:port``
+
+      See :ref:`AMQP Service for Bucket Notifications
+      <minio-server-envvar-bucket-notification-amqp>` for complete documentation
+      on each environment variable.
+
+   .. tab:: Configuration Settings
+
+      MinIO supports adding or updating AMQP endpoints on a running 
+      :mc:`minio server` process using the :mc-cmd:`mc admin config set` command 
+      and the :mc-conf:`notify_amqp` configuration key. You must restart the 
+      :mc:`minio server` process to apply any new or updated configuration
+      settings.
+
+      The following example code sets *all*  settings related to configuring an
+      AMQP service endpoint. The minimum *required* setting is 
+      :mc-conf:`notify_amqp url <notify_amqp.url>`:
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc admin config set ALIAS/ notify_amqp:IDENTIFIER \
+           url="ENDPOINT" \
+           exchange="<string>" \
+           exchange_type="<string>" \
+           routing_key="<string>" \
+           mandatory="<string>" \
+           durable="<string>" \
+           no_wait="<string>" \
+           internal="<string>" \
+           auto_deleted="<string>" \
+           delivery_mode="<string>" \
+           queue_dir="<string>" \
+           queue_limit="<string>" \
+           comment="<string>"
+
+      - Replace ``IDENTIFIER`` with a unique descriptive string for the
+        AMQP service endpoint. The following examples in this procedure
+        assume an identifier of ``primary``.
+
+        If the specified ``IDENTIFIER`` matches an existing AMQP service
+        endpoint on the MinIO deployment, the new settings *override* 
+        any existing settings for that endpoint. Use 
+        :mc-cmd:`mc admin config get notify_amqp <mc admin config get>` to
+        review the currently configured AMQP endpoints on the MinIO deployment.
+
+      - Replace ``ENDPOINT`` with the URL of the AMQP service endpoint.
+        For example:
+
+        ``amqp://user:password@hostname:port``
+
+      See :ref:`AMQP Bucket Notification Configuration Settings
+      <minio-server-config-bucket-notification-amqp>` for complete 
+      documentation on each setting.
+
+2) Restart the MinIO Deployment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You must restart the MinIO deployment to apply the configuration changes. 
+Use the :mc-cmd:`mc admin service restart` command to restart the deployment.
+
+.. important::
+
+   MinIO restarts *all* :mc:`minio server` processes associated to the 
+   deployment at the same time. Applications may experience a brief period of 
```

Comment:
Not sure if we need to say application will experience interruption due to the fact that it is only for a few seconds and applications are built to retry so this should be invisible to the application.

Reply:
So I wasn't sure if the majority of S3-API SDKs out there had built-in retry logic. I assumed at minimum they would throw errors or logs during downtime events, and figured it was worth mentioning in case users get surprised by a sudden burst of log messages during a restart.

If we *can* confidently state that applications transparently retry interrupted or 404'd operations, then I can tweak this slightly to add that.

Reply:
Addressed